lib/metadata.bash

```diff
@@ -20,6 +20,9 @@ function plugin_get_metadata() {
 function plugin_set_metadata() {
   local key="docker-compose-plugin-$1"
   local value="$2"
+
+  echo "buildkite-agent meta-data set $key $value";
+
```

Comment:
I assume these were debug lines and can be removed now, right?

Reply:
I think that's right.